src/isIsogram.test.js

```diff
@@ -6,4 +6,24 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it(`should return true for an empty string`, () => {
+    expect(isIsogram()).toBe(true);
+  });
+
+  it(`should return false for repeating letters`, () => {
+    expect(isIsogram('repeating')).toBe(false);
+  });
+
+  it(`should return false for repeating letters`, () => {
+    expect(isIsogram('RepeAting')).toBe(false);
```

Comment:
```suggestion
  it(`should return false for repeating letters`, () => {
    expect(isIsogram('repeating')).toBe(false);
  });

  it(`should return false for repeating letters`, () => {
    expect(isIsogram('RepeAting')).toBe(false);
```
you have the same test name but different logic of assertion. Try to think how to name your tests in an understandable way